From 15503b672487ecf1ce2d19b0b48c1d29544b93c8 Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Tue, 2 Feb 2021 09:43:51 -0800
Subject: [PATCH 01/17] Drop mistaken global variable definition in sdmmc_defs

which looks like it was meant to be a typedef, but the
code actually uses the struct tag in all declarations,
so dropping the type name is OK.

Signed-off-by: Matt Madison <matt@madison.systems>

Upstream-Status: Pending
---
 bootloader/partner/common/drivers/sdmmc/tegrabl_sdmmc_defs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bootloader/partner/common/drivers/sdmmc/tegrabl_sdmmc_defs.h b/bootloader/partner/common/drivers/sdmmc/tegrabl_sdmmc_defs.h
index 0130ad8..77e86fc 100644
--- a/bootloader/partner/common/drivers/sdmmc/tegrabl_sdmmc_defs.h
+++ b/bootloader/partner/common/drivers/sdmmc/tegrabl_sdmmc_defs.h
@@ -224,7 +224,7 @@ struct tegrabl_sdmmc {
 	bnum_t last_xfer_blocks;
 	void *last_xfer_buf;
 
-} sdmmc_context_t;
+};
 
 #define SDMMC_BLOCK_SIZE_LOG2			9U	/* 512 bytes */
 
-- 
2.34.1

